File: Calculator/src/Calculator.java
public class Calculator {

    public int addition(int... num) {
        int sum = 0;
        for (int add : num) {
            sum += add;
        }
        return sum;
    }
    public int subtraction(int... num) {
        int sub = 0;
        for (int minus : num) {
            sub -= minus;

        }
        return sub;
    }
    public int decimal(float... num) {
         int sum = 0;
        for (float add : num) {
            sum += add;
        }
            return sum;
    }

}

File: .idea/calculator/calculator.py
class Calculator:

    def addition(self, *args):
        total = 0
        for number in args:
            total += number
        return total

    def subtraction(self, number_one, number_two):
        return number_one - number_two

    def adbbby(self, number_one, number_two):
        return sum(number_one, number_two)

    def raise_value_error(self, *args):
        return ValueError

    def gets_float(self, number_one, number_two):
        return number_one + number_two






